open-api/rest-catalog-open-api.yaml

```diff
@@ -4458,7 +4458,9 @@ components:
     #  The fields `message` and `type` as indicated here are not presently prescriptive.
     UnauthorizedResponse:
       description:
-        Unauthorized. Authentication is required and has failed or has not yet been provided.
+        Unauthorized. The REST Catalog SHOULD respond with the 401 UnauthorizedResponse when
+        the access token provided is expired, revoked, malformed, or invalid for other reasons.
```

Comment:
If an access token is malformed, wouldn't a 400 / BadRequestErrorResponse be a more appropriate response?   I'm wondering if we should drop the `malformed` here?

Reply:
Hi @mrcnc thank you for the review!

IHMO I think if the access token is malformed, I'd still consider 401 to be the appropriate respones type vs 400 which I think would be more appropriate when the format of the Request itself is malformed.

I took this verbiage directly from https://www.rfc-editor.org/rfc/rfc6750 under the `invalid_token` section:

```
invalid_token
         The access token provided is expired, revoked, malformed, or
         invalid for other reasons.  The resource SHOULD respond with
         the HTTP 401 (Unauthorized) status code.  The client MAY
         request a new access token and retry the protected resource
         request.
```